lte/gateway/release/BUILD.bazel

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_pkg//pkg:deb.bzl", "pkg_deb")
+load("@rules_pkg//pkg:mappings.bzl", "pkg_attributes", "pkg_files")
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+
+SCTPD_PKGNAME = "magma-sctpd"
+
+VERSION = "1.8.0"
+
+ARCH = "amd64"
+
+FILE_NAME = "{name}_{ver}_{arch}".format(
+    name = SCTPD_PKGNAME,
+    arch = ARCH,
+    ver = VERSION,
+)
+
+genrule(
+    name = "gen_scptd_version",
```

Comment:
Can we fix the spelling here and in the other targets, it should be "sctpd" instead of "scptd"?

Reply:
o0 wups, sure - good catch

Reply:
fixed